db/schema.rb

```diff
@@ -142,8 +142,10 @@
   create_table "councillor_contributions", force: :cascade do |t|
     t.integer  "contributor_id", limit: 4
     t.integer  "authority_id",   limit: 4
-    t.datetime "created_at",               null: false
-    t.datetime "updated_at",               null: false
+    t.datetime "created_at",                                   null: false
+    t.datetime "updated_at",                                   null: false
+    t.text     "source",         limit: 65535
```

Comment:
This 'source' column is isn't in the migration on this PR, I think that's leaked in from another PR—got to be careful when running migrations and working across multiple branches to rollback any unrelated migrations before committing.